test/suite/wtdataset.py

```diff
@@ -47,6 +47,12 @@ def __init__(self, testcase, uri, rows, **kwargs):
         # If the timestamp generator is not set, get it from the test case.
         self.timestamp = kwargs.get('timestamp', testcase.getTimestamp())
 
+        # Get the tier populate share from the hook.
+        tier_share_percent = testcase.getTierSharePercent()
+        self.tier_share_value = 0
+        if tier_share_percent != 0 and self.rows != 0:
+            self.tier_share_value = int((int(tier_share_percent) / 100) * self.rows)
```

Comment:
By fixing the API to return an `int`, we can get rid of the inner `int(`.  You could even get rid of the outer `int(`, but using integer division `//` `instead of `/`, which returns a double`, and by reordering the operations:
```
   ....share_value = (tier_share_percent * self.rows) / 100
```
You can also note that the enclosing `if` is superfluous.  If either of those values is 0, `tier_share_value` will be assigned 0.

Reply:
Fixed.